corehq/util/quickcache.py

```diff
@@ -290,7 +290,7 @@ def domains_for_user(request):
             CacheWithTimeout(
                 django_caches['locmem'],
                 timeout=memoize_timeout)]
-        if timeout != 0:
+        if timeout > 0:
```

Comment:
Can `timeout` ever be `None`? If so this would product different behavior (not sure if that matters or not)